### ISSUE TITLE
Empty checkbox fields won't save

### ISSUE THREAD

**Issue by jalendport** [craftcms/cms#3844]
### Description

When you have a checkbox field on an entry and none of the option(s) are selected, saving the entry results in a `Couldn’t save entry.` error. The checkbox field is highlighted and says `CheckboxField is invalid.`

Looking in the logs I see:
```
2019-02-14 11:48:13 [-][1][-][info][craft\services\Elements::saveElement] Element not saved due to validation error: Array
(
    [testField[0].options[0].misc] => Array
        (
            [0] => CheckboxField is invalid.
        )

)
```

I just upgraded to the latest version of Craft CMS this morning so 99% sure this was introduced in one of the last couple releases (3.1.8 +).

### Steps to reproduce

1. Create a checkbox field and add it to a section
2. Try to save entry without selecting any of the checkbox field(s)

### Additional info

- Craft version: Craft CMS 3.1.10
- PHP version: PHP 7.1.25
- Database driver & version: MySQL 5.5.5
- Plugins & versions: tested on a clean install with no plugins

**Comment by samhernandez** [comment]
I've replicated the error on a fresh installation of Craft 3.1.10, PHP 7.2.10, PostgreSQL 11.1

**Comment by jalendport** [comment]
Thanks @samhernandez and @brandonkelly 🙌

**Comment by brandonkelly** [comment]
Just released 3.1.11 with a fix for this.
